build/wix/subdirs.txt

```diff
@@ -0,0 +1,7 @@
+controllers
```

Comment:
Do you have plans to have scons generate this? I can easily see Linux-based developers forgetting to update this file.